winnow/constants.py

```diff
@@ -78,14 +78,22 @@
     # "[+25.980265]-": 25.980265     # Carbamylation and ammonia loss
 }
 
+# Each C is also treated as Cysteine with carbamidomethylation in Prosit.
 INVALID_PROSIT_TOKENS: list = [
-    "\\+25.98",
-    "UNIMOD:7",
-    "UNIMOD:21",
-    "UNIMOD:1",
-    "UNIMOD:5",
-    "UNIMOD:385",
-    # Each C is also treated as Cysteine with carbamidomethylation in Prosit.
+    "(+25.98)",
```

Comment:
Is it correct that `(+25.98)` needs to be on brackets while the others (`+0.984`,  ` +42.011` , `+43.006`, `-17.027` ) are without brackets?